versions.yaml

```diff
@@ -45,7 +44,7 @@ versions:
       - '9.6.10'
```

Comment:
I think we should add suffix `-debian9` to all of tags (only `latest` can stay without).

Sample: https://github.com/GoogleCloudPlatform/rabbitmq-docker/blob/master/versions.yaml

Reply:
+1